Mobile/ShopLens/Droid/IO/ShopLensPictureDirectoryCreator.cs

```diff
@@ -14,11 +14,7 @@ private void CreateDirectoryForPictures(File pictureDirectory)
         {
             if (!pictureDirectory.Exists())
             {
-                if (!pictureDirectory.Mkdirs())
-                {
-                    // TO DO: throw valid exception.
-                    throw new System.Exception();
```

Comment:
We don't need to throw any exception here, because every single time the program will either use an already existing directory with the same name or create a new directory.